src/guides/collator_deployment.md

```diff
@@ -0,0 +1,182 @@
+  Network ID:        substrate
+  Secret seed:       0x79613c5ccb1c8aa9c21e1ddb2fbf65c8e979649cd52cfe041391c9f9d07ec3e9
+  Public key (hex):  0x4a503b20b3ada9ff3bf985271b5756e650a83ae47e44a7552dc2415ce8060a52
+  Account ID:        0x4a503b20b3ada9ff3bf985271b5756e650a83ae47e44a7552dc2415ce8060a52
+  Public key (SS58): 5Dk9EY7iUbjLX6fTDnbFZVGdnN7v2NgQtxj4Y7YjjZXgpCjE
+  SS58 Address:      5Dk9EY7iUbjLX6fTDnbFZVGdnN7v2NgQtxj4Y7YjjZXgpCjE
+```
+
+Store the secret phrase in a secure place. This account will be used on PolkadotJS app to bind you collator session keys to it.
+
+## 2- Generate the Node Keys
+This key is used in `libp2p` for secure communication between nodes in a peer to peer network. It's also know as a `PeerID`, it's unique across the network and is verifiable. The private part of the key should be kept secret while the public key is passed to other peers to enable them decrypt messages from the node.
+
+A Peer ID can be encoded into a multiaddr as a /p2p address with the Peer ID as a parameter. If my Peer ID is `QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N`, a libp2p multiaddress for me would be:
+```bash
+/p2p/QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N
+```
+
+As with other multiaddrs, a `/p2p` address can be encapsulated into another multiaddr to compose into a new multiaddr. For example, I can combine the above with a transport address `/ip4/198.51.100.0/tcp/4242` to produce this very useful address:
+```bash
+/ip4/198.51.100.0/tcp/4242/p2p/QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N
+```
+
+This provides enough information to dial a specific peer over a TCP/IP transport. If some other peer has taken over that IP address or port, it will be immediately obvious, since they will not have control over the key pair used to produce the Peer ID embedded in the address.
+
+When a blockchain node run, this key will be generated automatically. However, it's important to generate static keys for bootnodes and collators. This is due to the fact that these nodes are specified in the chainspec with their corresponding keys, so other nodes when connect to them can verify their identity before downloading the blocks.
+
+In order to generate the node key from the parachain binary:
+```bash
+> parachain-template-node key generate-node-key --file node.key
+# example output
+12D3KooWExcVYu7Mvjd4kxPVLwN2ZPnZ5NyLZ5ft477wqzfP2q6E # PeerId (hash of public node key)
+cat node.key
+d5e120e30dfb0eac39b1a6727d81c548e9c6b39ca97e565438a33d87726781a6% # private node key, do not copy the percent sign
+```
+The keys then will be passed to the parachain binary as an argument with `--node-key <path to the file>`.
+
+Note that if your collator nodes are not mentioned in the chainspec, you can simply skip this step! This is due to the fact that nodes always connect to chainspec collators first (or bootnodes on the relaychain) and from there they discover other nodes in the network so it's vital to be able to verify the intial collectors' identity. 
```

Comment:
I would prefer to remove this sentence completely. It is only valid if your collator is also a bootnode. But the goal of this guide is simply to add an extra collator to the network so it would be better not to confuse the readers with bootnode related config.